lightkurve/targetpixelfile.py

```diff
@@ -1012,23 +1014,82 @@ def from_fits_images(images, position=None, size=(10, 10), extension=None,
                                                n_rows=size[0],
                                                n_cols=size[1],
                                                target_id=target_id)
+
+        mid_img = images[int(len(images)/2)-1] # Find middle image to use as a reference
+        #Check if the image is a single FITS image, or a FITS file and get the appropriate PrimaryHDU or ImageHDU,
+        if isinstance(mid_img, fits.ImageHDU):
+            mid_hdu = mid_img
+        elif isinstance(mid_img, fits.HDUList):
+            mid_hdu = mid_img[extension]
+        else:
+            mid_hdu = fits.open(mid_img)[extension]
+        # Calculate reference WCS coords from the middle image. Get the middle pixel column and row.
+        wcs = WCS(mid_hdu)
+        pixelpos = skycoord_to_pixel(position, wcs=wcs)
+        column, row = int(np.round(pixelpos[0])), int(np.round(pixelpos[1]))
+
+        # Calculate the ra and dec of the center pixel of the middle image
+        skyposctr = pixel_to_skycoord(column, row, wcs=wcs)
+        ractr = skyposctr.ra.deg
+        decctr = skyposctr.dec.deg
```

Comment:
Ah, good point- this was originally put in by me. The reason was that when I cut out the tpf, the object of interest was not necessarily in the middle of the center pixel. We need the RA and Dec of this location, and it can differ from RA_OBJ/DEC_OBJ by a couple arcsec, due to the large Kepler pixel size. Does that make sense? Now of course we are using the astropy cutout function, and I admittedly did not check whether puts the star in the middle of the center pixel and regrids the coordinates. Unless it does that, then we still need these few lines to get the exact center of the TPF for the WCS solution.

Reply:
Hmmm, does this really matter?  As long as we have the (ra, dec) for a specific pixel coordinate near the center, we can do the cutout and a wcs -- i.e. can we get rid of it?  Less code is better =)

...or did I misunderstand?

Reply:
Well, my understanding is that we need a reference point from which to define the WCS in both world and image coordinates. I was selecting that point to be the center of the TPF, which is not the same as the object location. Thus this bit of code calculates the RA and Dec of the center, so that we can put it in the header.